go.mod

```diff
@@ -7,8 +7,11 @@ require (
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
 	github.com/coreos/ignition v0.35.0
 	github.com/coreos/ignition/v2 v2.15.0
+	github.com/go-logr/stdr v1.2.2
 	github.com/google/go-cmp v0.5.9
 	github.com/jaypipes/ghw v0.8.1-0.20210605191321-eb162add542b
+	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.1.1-0.20201119153432-9d213757d22d
+	github.com/k8snetworkplumbingwg/sriov-network-operator v1.2.0
```

Comment:
can this be a issue when we integrate the tests into cnf-tests?

Reply:
if you mean cnf-tests image, we do not integrate tests there anymore apart from the latency suite

Reply:
Uhm, right